resources/functionMap_bleedingEdge.php

```diff
@@ -107,6 +107,7 @@
 		'ImagickKernel::scale' => ['void', 'scale'=>'float', 'normalizeFlag'=>'Imagick::NORMALIZE_KERNEL_*'],
 		'max' => ['', '...arg1'=>'non-empty-array'],
 		'min' => ['', '...arg1'=>'non-empty-array'],
+		'file' => ['list<string>|false', 'filename'=>'string', 'flags='=>'0|FILE_USE_INCLUDE_PATH|FILE_IGNORE_NEW_LINES|FILE_SKIP_EMPTY_LINES', 'context='=>'resource'],
```

Comment:
This seems to be missing the `FILE_NO_DEFAULT_CONTEXT` flag.

Reply:
thanks will have a look

Reply:
added via https://github.com/phpstan/phpstan-src/pull/2482

I missed it, because its missing from https://www.php.net/manual/en/function.file.php - therefore just also [sumbitted a php.net docs PR](https://github.com/php/doc-en/pull/2540)